`feature_block` has failed:
```bash
2020-12-02T04:37:36.022000Z TestFramework (INFO): Accept a block with invalid opcodes in dead execution paths
2020-12-02T04:37:36.134000Z TestFramework (INFO): Test re-orging blocks with OP_RETURN in them
2020-12-02T04:37:36.912000Z TestFramework (INFO): Test a re-org of one week's worth of blocks (1088 blocks)
2020-12-02T04:46:04.530000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 126, in main
    self.run_test()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_block.py", line 1278, in run_test
    self.send_blocks([block], True, timeout=2440)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_block.py", line 1410, in send_blocks
    self.helper_peer.send_blocks_and_test(blocks, self.nodes[0], success=success, reject_reason=reject_reason, force_send=force_send, timeout=timeout, expect_disconnect=reconnect)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 631, in send_blocks_and_test
    self.sync_with_ping(timeout=timeout)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 507, in sync_with_ping
    self.wait_until(test_function, timeout=timeout)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 412, in wait_until
    wait_until_helper(test_function, timeout=timeout, lock=p2p_lock, timeout_factor=self.timeout_factor)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/util.py", line 247, in wait_until_helper
    if predicate():
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 409, in test_function
    assert self.is_connected
AssertionError
2020-12-02T04:46:06.683000Z TestFramework (INFO): Stopping nodes
stderr:
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/authproxy.py", line 107, in _request
    self.__conn.request(method, path, postdata, headers)
  File "/usr/lib/python3.8/http/client.py", line 1255, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1250, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1049, in _send_output
    self.send(chunk)
  File "/usr/lib/python3.8/http/client.py", line 971, in send
    self.sock.sendall(data)
BrokenPipeError: [Errno 32] Broken pipe
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_block.py", line 1417, in <module>
    FullBlockTest().main()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 149, in main
    exit_code = self.shutdown()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 278, in shutdown
    self.stop_nodes()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 526, in stop_nodes
    node.stop_node(wait=wait)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py", line 319, in stop_node
    self.stop(wait=wait)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/coverage.py", line 47, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/authproxy.py", line 144, in __call__
    response, status = self._request('POST', self.__url.path, postdata.encode('utf-8'))
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/authproxy.py", line 113, in _request
    self.__conn.request(method, path, postdata, headers)
  File "/usr/lib/python3.8/http/client.py", line 1255, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1250, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1010, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 950, in send
    self.connect()
  File "/usr/lib/python3.8/http/client.py", line 921, in connect
    self.sock = self._create_connection(
  File "/usr/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/usr/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
```

```bash
 node0 2020-12-02T04:44:06.247207Z [msghand] - Disconnect block: 2249.91ms 
 node0 2020-12-02T04:44:06.863029Z [msghand] UpdateTip: new best=107d0d7d666cc30a086b3346a220e0f71fe736988247e8c7227b7025c92e7993 height=378 version=0x00000004 log2_work=9.566054 tx=11656 date='2020-12-02T04:42:46Z' progress=1.000000 cache=1.6MiB(12001txo) 
 node0 2020-12-02T04:44:06.863218Z [msghand] Enqueuing BlockDisconnected: block hash=621dba2ca71816fc9d2e28994582593ea03aa4184638ee3962a7ed632e8b2ba8 block height=379 
 node0 2020-12-02T04:44:18.793214Z [msghand] - Disconnect block: 7207.89ms 
 node0 2020-12-02T04:44:53.834226Z [msghand] UpdateTip: new best=73b411695b1980e6ea88ae9fd02cddd51a71be59e988eee78243ffd0eaf60ccf height=377 version=0x00000004 log2_work=9.562242 tx=11653 date='2020-12-02T04:42:45Z' progress=1.000000 cache=1.6MiB(12001txo) 
 node0 2020-12-02T04:45:01.905174Z [msghand] Enqueuing BlockDisconnected: block hash=107d0d7d666cc30a086b3346a220e0f71fe736988247e8c7227b7025c92e7993 block height=378 
 test  2020-12-02T04:46:03.936000Z TestFramework.p2p (DEBUG): Closed connection to: 127.0.0.1:16244 
 test  2020-12-02T04:46:04.530000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 126, in main
                                       self.run_test()
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_block.py", line 1278, in run_test
                                       self.send_blocks([block], True, timeout=2440)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_block.py", line 1410, in send_blocks
                                       self.helper_peer.send_blocks_and_test(blocks, self.nodes[0], success=success, reject_reason=reject_reason, force_send=force_send, timeout=timeout, expect_disconnect=reconnect)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 631, in send_blocks_and_test
                                       self.sync_with_ping(timeout=timeout)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 507, in sync_with_ping
                                       self.wait_until(test_function, timeout=timeout)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 412, in wait_until
                                       wait_until_helper(test_function, timeout=timeout, lock=p2p_lock, timeout_factor=self.timeout_factor)
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/util.py", line 247, in wait_until_helper
                                       if predicate():
                                     File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/p2p.py", line 409, in test_function
                                       assert self.is_connected
                                   AssertionError
 test  2020-12-02T04:46:06.528000Z TestFramework (DEBUG): Closing down network thread 
 test  2020-12-02T04:46:06.683000Z TestFramework (INFO): Stopping nodes 
 test  2020-12-02T04:46:06.683000Z TestFramework.node0 (DEBUG): Stopping node 
```

I've added a commit to increase the memory of the container to the [allowed maximum](https://cirrus-ci.org/guide/linux/#linux-containers). Either that fixes it, or we update the comment to make it generic, as Travis is going away, and this is currently being run on Cirrus.

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -92,8 +92,8 @@ task:
   << : *GLOBAL_TASK_TEMPLATE
   container:
     image: ubuntu:focal
-    cpu: 4  # Double CPU and Memory to avoid timeout
-    memory: 16G
+    cpu: 4  # Double CPU and increase Memory to avoid timeout
+    memory: 24G
   env:
     MAKEJOBS: "-j8"
     FILE_ENV: "./ci/test/00_setup_env_native_tsan.sh"

--- a/ci/test/00_setup_env_native_tsan.sh
+++ b/ci/test/00_setup_env_native_tsan.sh
@@ -10,6 +10,5 @@ export CONTAINER_NAME=ci_native_tsan
 export DOCKER_NAME_TAG=ubuntu:20.04
 export PACKAGES="clang llvm libc++abi-dev libc++-dev python3-zmq"
 export DEP_OPTS="CC=clang CXX='clang++ -stdlib=libc++'"
-export TEST_RUNNER_EXTRA="--exclude feature_block"  # Low memory on Travis machines, exclude feature_block.
 export GOAL="install"
 export BITCOIN_CONFIG="--enable-zmq --with-gui=no CPPFLAGS='-DARENA_DEBUG -DDEBUG_LOCKORDER' CXXFLAGS='-g' --with-sanitizers=thread CC=clang CXX='clang++ -stdlib=libc++' --with-boost-process"

